### ISSUE TITLE
CSV Download with Field Labels (Instead of API Names) and Korean characters support

### ISSUE THREAD

**Issue by argenturatbek** [tomansley/SimpliListViews#54]
**Is your feature request related to a problem? Please describe.**
Hi Tom. Today during our weekly meeting, I introduced the CSV download option to the team. They really liked this functionality of the Simpli List View app. But the problem (maybe not a problem) is that the records that have Korean characters are not displayed in the downloaded CSV file. That's first. The second feature request would be to include field label names instead of field API names. 

**Describe the solution you'd like**
Since our customers are not well familiar with API names, I would like see Field Label names. And the most of our fields are in Korean although there some English word variations.

**Additional context**
![image](https://user-images.githubusercontent.com/59495051/141921966-0abeee31-8613-4b37-a4fd-bd73644d7624.png)
![image](https://user-images.githubusercontent.com/59495051/141922057-eab6a8a3-4bb0-402c-98e3-924b2538ab04.png)

**Comment by tomansley** [comment]
Hi Argen,

Did you find a way around these issues or is it no longer a request?

Cheers
Tom

**Comment by argenturatbek** [comment]
Hi Tom, 

I hope you are doing great. It seems like our client may not need this functionality anymore. Based on what my colleague said, they are going to use Salesforce reports for this purpose. Therefore, I decided to close this issue so as not to take your time. Oh and, I did not find any workaround regarding this. I will let you know when I do so!

As always, thank you, Tom!

Best. Argen

**Comment by tomansley** [comment]
Thanks for letting me Argen. I might wait on someone else needing language support but I will go ahead and make changes for the header labels. I think it makes sense. 


Just an FYI. 


On Nov 29, 2021 19:15, Argen Turatbek ***@***.***> wrote: 

Hi Tom,

I hope you are doing great. It seems like our client may not need this functionality anymore. Based on what my colleague said, they are going to use Salesforce reports for this purpose. Therefore, I decided to close this issue so as not to take your time. Oh and, I did not find any workaround regarding this. I will let you know when I do so!

As always, thank you, Tom!

Best. Argen

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
